pkg/auth/model/model.go

```diff
@@ -41,23 +41,24 @@ type User struct {
 }
 
 // hashPassword generates a hashed password from a plaintext string
-func hashPassword(password string) ([]byte, error) {
-	pw, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
-	if err != nil {
-		return nil, err
-	}
-	return pw, nil
-}
 
 func (u *User) UpdatePassword(password string) error {
-	pw, err := hashPassword(password)
+	pw, err := HashPassword(password)
 	if err != nil {
 		return err
 	}
 	u.EncryptedPassword = pw
 	return nil
 }
 
+func HashPassword(password string) ([]byte, error) {
+	pw, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
+	if err != nil {
+		return nil, err
+	}
+	return pw, nil
```

Comment:
```suggestion
	return bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
```

Reply:
Not relevant to the new function created.

Reply:
I'm not following this comment thread.  Isn't the new function _exactly_ the single line in the comment above?

Reply:
No, here we must check for errors before we transact with the DB.